hera_cal/lstbin.py

```diff
@@ -111,8 +118,12 @@ def lst_bin(data_list, lst_list, flags_list=None, dlst=None, begin_lst=None, lst
     # themselves hold lists of ndarrays
     data = odict()
     flags = odict()
+    nsamples = odict()
     all_lst_indices = set()
-
+    pols = []
```

Comment:
I'd do `list(set([pol for dc in data_list for pol in dc.pols()]))`

Reply:
I've adopted this.